### DISCUSSION TITLE
Does Copilot Chat for VS Code have search or bookmarks?

### DISCUSSION THREAD

**Discussion by pietmondrian** [community/community#106836] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I am using Copilot Chat a lot, so far in a single thread.  Sometimes I want to go back to a conversation from a few days back.  I have to manually scroll and look for it.  I wish Copilot Chat would have more bookmarking and search capabilities.  

How have you solved this information management problem?  Thank you!

**Comment by baccarani** [comment, marked as answer]
This functionality is currently not supported on VS Code Copilot Chat. However, this would be a great enhancement as this feature is available on ChatGPT and Bing Chat.

**Comment by pietmondrian** [comment]
Thank you, @baccarani.  How do you manage the large amount of Q&A? Do you start different threads?  I tried to put all my conversations into Google Docs or Word.  But it is too much work given how much conversation volume piles up daily.  I tried using ChatGPT but it is not integrated into VSC. I wish they added more editing features like comments, searching, bookmarking, and organizing the content.
